languages/python/sqlalchemy-oso/sqlalchemy_oso/compat.py

```diff
@@ -0,0 +1,25 @@
+"""SQLAlchemy version compatibility tools.
+
+Keep us compatible with 1.3 and 1.4 by implementing wrappers when needed here.
+"""
+from packaging import version
+
+import sqlalchemy
+
+AT_LEAST_SQLALCHEMY_VERSION_1_4 = version.parse(
+    sqlalchemy.__version__  # type: ignore
+) >= version.parse("1.4")
```

Comment:
Not sure if there's a better way to do this, but SQLAlchemy doesn't seem to export a Django-style `VERSION` constant.